integrations/static_analyzers/entrypoint.sh

```diff
@@ -3,9 +3,10 @@ mkdir -p ${LOG_PATH}
 touch ${LOG_PATH}/gunicorn_access.log ${LOG_PATH}/gunicorn_errors.log
 chown -R static_analyzers-user:static_analyzers-user ${LOG_PATH}
 su static_analyzers-user -s /bin/bash
+freshclam && clamd
```

Comment:
```suggestion
# start clam service itself and the updater in background as daemon
freshclam -d &
clamd &

```
so they both run in the background.

Reply:
I think `clamd` starts in the background only by default and `-d` option for `freshclam` makes it start as a daemon. So,
```bash
freshclam -d
clamd
```
should be enough I think